File: DMOJ/DMOJ/Mispelling/Mispelling.cs
using System;
class Mispelling
{
	public static void Main(string[] args)
	{
		int numLines = Int32.Parse(Console.ReadLine());
		string[] arr = new string[numLines];
		string[] temp = new string[2];
		string j;
		int k;
		for (int i = 0; i < numLines; i++)
		{
			j = Console.ReadLine();
			temp = j.Split(" ",2);
			k = Int32.Parse(temp[0]-1);
			arr[i] = temp[1].Remove(k,1);
		}

		for (int i = 0; i < arr.Length; i++)
			Console.WriteLine((i+1) + " " + arr[i]);
	}
}

File: DMOJ/DMOJ/Mispelling/Mispelling.java
#import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
public class Mispelling
{
	public static void main(String[] args) throws IOException
	{
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		int n = Integer.parseInt(br.readLine()), i = 0;
		while (i++ < n) {
			StringBuilder sb = new StringBuilder(br.readLine());
			int remove = sb.indexOf(" ");
			sb.deleteCharAt(Integer.parseInt(sb.substring(0,remove)) + remove);
			sb.delete(0, remove);
			sb.insert(0,i);
			System.out.println(sb.toString());
		}
		br.close();
		return;
	}
}
